website/docs/guides/best-practices/how-we-style/1-how-we-style-our-dbt-models.md

```diff
@@ -0,0 +1,17 @@
+---
+title: How we style our dbt models
+id: 1-how-we-style-our-dbt-models
+---
+
+## Fields and model names
```

Comment:
For model naming, I did like the old "All objects should be plural" from our previous style guide. I've had customers run into snags a couple of times in RO's because they weren't sure if the model name was, e.g. `stg_payment` or `stg_payments`, so I think it's a good thing to standardize.

Reply:
Another good one from the old guide is limiting abbreviations in model and column names for readability.